### ISSUE TITLE
OpenVPN failure

### ISSUE THREAD

**Issue by thypon** [voidlinux/void-packages#1316]
```
Tue Apr 14 15:32:45 2015 /usr/sbin/ip link set dev tap0 up mtu 1500
Tue Apr 14 15:32:45 2015 Linux ip link set failed: could not execute external program
Tue Apr 14 15:32:45 2015 Exiting due to fatal error
```
```
[gattuso@R2D2 ~]$ whereis ip
ip: /usr/bin/ip /usr/share/man/man8/ip.8 /usr/share/man/man7/ip.7
```

**Comment by bougyman** [comment]
There is more scroll before that ip link failure. Perhaps you don't have tun?

`sudo modprobe tun`

And try again

**Comment by xtraeme** [comment]
Fixed in `openvpn-2.3.6_6`.

**Comment by thypon** [comment]
@bougyman ip was in a different place, not expected from openvpn.
